specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/EventGrid.json

```diff
@@ -306,6 +306,13 @@
       "description": "Version of the API to be used with the client request.",
       "required": true,
       "type": "string"
+    },
+	"ChannelNameParameter": {
```

Comment:
You need to add a ref to the parameter from the method that needs it, here:
https://github.com/Azure/azure-rest-api-specs/blob/dc439efcfc15448824877603f66fc1578d1c71c5/specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/EventGrid.json#L97-L99